proto/poktroll/tokenomics/params.proto

```diff
@@ -14,19 +14,19 @@ message Params {
 
   // mint_allocation_dao is the percentage of the minted tokens which are sent
   // to the DAO reward address during claim settlement.
-  double mint_allocation_dao = 1 [(gogoproto.jsontag) = "mint_allocation_dao", (gogoproto.moretags) = "yaml:\"mint_allocation_dao\""];
+  double mint_allocation_dao = 1 [(gogoproto.jsontag) = "mint_allocation_dao", (gogoproto.moretags) = "yaml:\"mint_allocation_dao\", disable_individual_update:\"true\""];
```

Comment:
While I did make this change because of the test, I feel that it's still appropriate that these fields are able to indicate that they cannot be updated individually. This constraint is novel and will not apply to all tokenomics module params (i.e. `dao_reward_address` will be individually updatable).

Reply:
UPDATE: Upon further reflection, I've realized that we can exclude these fields from the param update test by changing the `ValidParams` fields to be pointers, then making the non-updatable fields `nil` in `TokenomicsModuleParamConfig#ValidParams`. :sweat_smile: